telco-hub/configuration/reference-crs-kube-compare/required/gitops/addPluginsPolicy.yaml

```diff
@@ -0,0 +1,87 @@
+---
+# This content creates a policy which installs the necessary argocd
+# plugins.
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  annotations:
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+    policy.open-cluster-management.io/description: ""
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    argocd.argoproj.io/sync-wave: "10"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+  name: ztp-argocd-plugins-installer
+  namespace: hub-policies
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: ztp-argocd-plugins-cp
+        spec:
+          remediationAction: enforce
+          severity: high
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: argoproj.io/v1beta1
+                kind: ArgoCD
+                metadata:
+                  name: openshift-gitops
+                  namespace: openshift-gitops
+                spec:
+                  controller:
+                    resources:
+                      limits:
+                        cpu: "16"
+                        memory: 32Gi
```

Comment:
We should allow the user to increase, but not decrease these. ie check for "at least". Same for other limits below.

Reply:
I didn't find a way to make flexible those fields that are so nested into the policy. Had to restrict the enforcement in the policy.